Hypnotic-Herons/bangazon-api
# Dv computer

## Link to Ticket
Closes  https://github.com/Hypnotic-Herons/bangazon-api/issues/8


## Description of Proposed Changes
Added files to create a table for computer in the db and manage that data
- 

## Steps to Test

Outline the steps to test

```
git fetch --all
git checkout dvComputer
code .

```


1. python manage.py makemigrations api
1. python manage.py migrate
1. python manage.py runserver
1. view in browser to see the api root
1.  add departments and employees for testing
1. add computer and associate with an employee (purch date is required)
1. check post, put, get and delete
  

## Impacted Areas in Application

List general components of the application that this PR will affect:

* data structure for the computer table and associations to employee table

## Mentions @username

@Levi-Schubert @MatthewKelly12 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
